guides/source/caching_with_rails.md

```diff
@@ -253,7 +254,7 @@ This is the default cache store implementation.
 
 This cache store uses Danga's `memcached` server to provide a centralized cache for your application. Rails uses the bundled `dalli` gem by default. This is currently the most popular cache store for production websites. It can be used to provide a single, shared cache cluster with very high performance and redundancy.
 
-When initializing the cache, you need to specify the addresses for all memcached servers in your cluster. If none is specified, it will assume memcached is running on the local host on the default port, but this is not an ideal set up for larger sites.
+When initializing the cache, you need to specify the addresses for all memcached servers in your cluster. If none are specified, it will assume memcached is running on localhost on the default port, but this is not an ideal setup for larger sites.
```

Comment:
AFAIK, because "addresses" is plural it is acceptable to follow none with either a singular or a plural verb. I personally find "none is" to read awkwardly, which is why I changed it. If you disagree, I'll gladly revert it. :smile: 

For reference, from the OED: "It is sometimes held that none can only take a singular verb, never a plural verb: none of them is coming tonight rather than none of them are coming tonight. There is little justification, historical or grammatical, for this view. None is descended from Old English nān meaning ‘not one’ and has been used for around a thousand years with both a singular and a plural verb, depending on the context and the emphasis needed."